R/applyFilt.R

```diff
@@ -140,76 +147,63 @@ applyFilt <- function (filter_object, omicsData, ...) {
 
   # generate warnings if data type is not present but user asks to filter #
   if (!is.null(filter_object$filter_edata) & is.null(edata_cname)) {
-
-    warning (paste("e_data identifier column specified in filter_object is not",
-                   "present in omicsData$e_data. Specified e_data features",
-                   "cannot be filtered from data.",
-                   sep = " "))
-
+    warning(paste("e_data identifier column specified in filter_object is not",
+      "present in omicsData$e_data. Specified e_data features",
+      "cannot be filtered from data.",
+      sep = " "
+    ))
   }
 
   if (!is.null(filter_object$filter_emeta) & is.null(emeta_cname)) {
-
-    warning (paste("e_meta identifier column specified in filter_object is not",
-                   "present in omicsData$e_meta. Specified e_meta features",
-                   "cannot be filtered from data.",
-                   sep = " "))
-
+    warning(paste("e_meta identifier column specified in filter_object is not",
+      "present in omicsData$e_meta. Specified e_meta features",
+      "cannot be filtered from data.",
+      sep = " "
+    ))
   }
 
   if (!is.null(filter_object$filter_samples) & is.null(samp_cname)) {
-
-    warning (paste("Sample identifier column specified in filter_object is not",
-                   "present in omicsData. Specified samples cannot be filtered",
-                   "from data.",
-                   sep = " "))
-
+    warning(paste("Sample identifier column specified in filter_object is not",
+      "present in omicsData. Specified samples cannot be filtered",
+      "from data.",
+      sep = " "
+    ))
   }
 
 
-  UseMethod ("applyFilt")
-
+  UseMethod("applyFilt")
 }
 
 # function for moleculeFilt
 #' @export
 #' @name applyFilt
 #' @rdname applyFilt
-applyFilt.moleculeFilt <- function(filter_object, omicsData, min_num=2){
-
+applyFilt.moleculeFilt <- function(filter_object, omicsData, min_num = 2, ...) {
```

Comment:
Yes, it fixes a warning (the base applyFilt function needs the ellipses parameter so that it can be extended, and extended functions need to have all of the parameters of the base).

Reply:
You can see of examples of this in base R libraries if you look at the `summary` help page.